wiki/osu!wiki_Contribution_Guide/Locally/en.md

```diff
@@ -0,0 +1,71 @@
+# Locally
+
+*Main page: [osu!wiki Contribution Guide](/wiki/owcg).*
+
+*This article continues from the main page* and assumes that you will be using [GitHub Desktop](https://desktop.github.com). You may use other git clients on your own terms. Even if you do work locally, you will still need to access GitHub to create pull requests to make your changes happen.
+
+## Installing GitHub Desktop
+
+You will need to then install [GitHub Desktop](https://desktop.github.com). See [Getting Started with GitHub Desktop](https://help.github.com/desktop-classic/guides/getting-started/) for help.
+
+When the installer is completed, GitHub Desktop will prompt you to login. Click on the `Sign into GitHub.com` link and enter your GitHub's username (or the email you used) and password. You will then be prompted to configure Git, just fill in the prompts and continue. On the next screen, it will prompt you if you want to submit anonymous usage data, it is your call if you want to opt-out or not.
+
+## Clone
+
+Once you are done setting up GitHub desktop, click on `Clone a repository`.
+
+![](img/github-desktop-clone.jpg)
+
+This will show a list of your repos; select the `osu-wiki` repo. The location of the cloned repo is up to you, but take note as to where you clone it to.
+
+![](img/github-desktop-select-repo.jpg)
+
+Once you have decided on the clone location, click `Clone`.
```

Comment:
add `this will take a few minutes (depending on your network environment).` in the end.
btw, osu-wiki repo is 300mb at least to download.